zeek-agent/framework/framework_commons.zeek

```diff
@@ -1,5 +1,14 @@
 module ZeekAgent;
 
+# Increase network session timeouts for better attribution
```

Comment:
I don't think we want to solve this problem like this.  Changing these numbers can have a number of side effects in terms of memory use and logging behavior.

Reply:
Better would be to have mechanisms in zeek to deal with the asynchronous nature of some data flows with regards to logging.  We've been thinking about a mechanism for "delayed logging" for quite a while that would help to address this sort of behavior.